frontend/packages/process-editor/src/components/ConfigPanel/ConfigPanel.test.tsx

```diff
@@ -1,32 +1,21 @@
 import React from 'react';
 import { ConfigPanel } from './ConfigPanel';
-import { render, screen } from '@testing-library/react';
+import { act, render, screen } from '@testing-library/react';
```

Comment:
We should avoid using `act`, since the last testing-library package was updated last week 😊 
```suggestion
import { render, screen } from '@testing-library/react';
```